src/test/ui/const-generics/issue-62879.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(const_generics)]
+//~^ WARN the feature `const_generics` is incomplete and may cause the compiler to crash
+
+fn foo<const N: usize, const A: [u8; N]>() {}
+//~^ ERROR: cycle detected when processing `foo::A`
```

Comment:
This test should be check-pass (cc @varkor), but a cycle error occurs due to a lack of lazy normalization. Please leave a comment noting this and file a replacement issue for this test becoming check-pass.

Reply:
Added a comment, and filed it as #68249.